src/styles/main.scss

```
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
+@import "reset";
+@import "mixins";
+@import "antisnake";
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
 }
```

Спочатку краще записати правила для селектора html
```suggestion
html {
  font-family: Arial, Helvetica, sans-serif;
  font-size: 100px;
}

@import "reset";
@import "mixins";
@import "antisnake";

```

---

переробив :)